tests/integration/components/slugged-route-model-details-test.js

```diff
@@ -1,17 +1,35 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 import { mockRouter } from 'code-corps-ember/tests/helpers/mock-routing';
+import PageObject from 'ember-cli-page-object';
+import component from 'code-corps-ember/tests/pages/components/slugged-route-model-details';
+
+let page = PageObject.create(component);
+
+function renderPage() {
+  page.render(hbs`
+    {{slugged-route-model-details
+      sluggedRoute=sluggedRoute
+      organization=sluggedRoute.organization
+      user=sluggedRoute.user
+    }}`);
+}
 
 moduleForComponent('slugged-route-model-details', 'Integration | Component | slugged route model details', {
   integration: true,
   beforeEach() {
+    page.setContext(this);
     mockRouter(this);
+  },
+  afterEach() {
+    page.removeContext();
   }
 });
 
 test('it renders', function(assert) {
-  this.render(hbs`{{slugged-route-model-details}}`);
-  assert.equal(this.$('.slugged-route-model-details').length, 1);
+  assert.expect(1);
+  renderPage();
+  assert.ok(page.isVisible, true, 'the component render\'s its element.');
 });
```

Comment:
I would remove this test altogether, since we're testing that implicitly elsewhere here.